modules/security-context-constraints-about.adoc

```diff
@@ -147,7 +147,7 @@ The `restricted` SCC:
 * Ensures that pods cannot mount host directory volumes
 * Requires that a pod is run as a user in a pre-allocated range of UIDs
 * Requires that a pod is run with a pre-allocated MCS label
-* Allows pods to use any FSGroup
+* Allows Pods to use FSGroup as per SCC strategies
```

Comment:
Can you please change "Pods" back to lowercase? 

And then I'm not sure what you mean by you can use the FSGroup "as per SCC strategies". Can you clarify what this is supposed to mean?

@stlaz Can you please weigh in on this as well?

```suggestion
* Allows pods to use the FSGroup as per SCC strategies
```

Reply:
Actually, I just came across another bug for this (OCPBUGS-30263). The requester suggested this wording:

```suggestion
* Requires that a pod is run with a preallocated FSGroup
```
Can you update to use that? @stlaz can you confirm that this wording is accurate?

Reply:
The latter narrows it down a bit and is more specific to the strategy of the `restricted` SCC. You can use the "preallocated" sentence.
I wonder if that might by any chance raise a question about how the group is being preallocated. I guess we have docs for that, too?

Reply:
Thanks @stlaz!

@s-dharma Can you please update to use this sentence instead? `Requires that a pod is run with a preallocated FSGroup`